cli/src/main/kotlin/Main.kt

```diff
@@ -141,4 +165,25 @@ object Main : CommandWithHelp() {
 
         println()
     }
+
+    private fun loadConfig(): OrtConfiguration {
+        val argConfig = ConfigFactory.parseMap(configArguments, "Command line").withOnlyPath("ort")
+        val systemConfig = ConfigFactory.systemProperties().withOnlyPath("ort")
+        val fileConfig = configFile?.expandTilde()?.let {
+            require(it.isFile) {
+                "The provided configuration file '$it' is not actually a file."
+            }
+
+            ConfigFactory.parseFile(it).withOnlyPath("ort")
+        }
+        val defaultConfig = ConfigFactory.parseResources("default.conf")
```

Comment:
Shouldn't we instead use `ConfigFactory.defaultApplication()` and rename our default configuration file to `application.conf`?

Reply:
That function does too much magic, I'd rather be in control of how we load the configuration, see: https://lightbend.github.io/config/latest/api/com/typesafe/config/ConfigFactory.html#defaultApplication--